buildSrc/src/main/java/org/springframework/boot/gradle/JavaConventionsPlugin.java

```diff
@@ -173,6 +173,11 @@ private void configureTestConventions(Project project) {
 				testLoggingContainer.setShowStackTraces(true);
 				testLoggingContainer.setExceptionFormat(TestExceptionFormat.FULL);
 			});
+			test.jvmArgs(
```

Comment:
@sobychacko this removes the warning when running tests in any project. 

Where else were you seeing the warning? I thought it was showing up when running the sample apps but I am unable to reproduce that now (I wonder if the DNS is cached and then not hitting that code path that triggers it). I have a fix queued up for the sample apps if I can prove it is happening there.

Reply:
@onobc I think I saw those warnings when running the sample apps.

Reply:
@sobychacko  Can you see if you can reproduce it?